app/views/shared/check_answers/_one_link_section.html.erb

```diff
@@ -1,40 +1,42 @@
 <% read_only = false unless local_assigns[:read_only] %>
 
-<div class="govuk-grid-row" id="app-check-your-answers__<%= name %><%= answer_hash.present? ? "" : "_header" %>">
-  <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-m"><%= question %></h2>
-  </div>
-  <% if answer_hash.present? && !read_only %>
-    <div class="govuk-grid-column-one-third govuk-summary-list--no-border align-text-right">
-      <p>
-        <%= govuk_link_to(t("generic.change"), url, class: "change-link", "aria-label": question) %>
-      </p>
+<section class="print-no-break">
```

Comment:
You're surrounding the heading and the first summary card both here, inside the partial, and outside the partial in the `_assets` partial, so on the frontend you'd be able to see two lots of `<section class="print-no-break">` and we'd only need one of them.

It might be easier to leave it just here in the partial, so that it only needs to be written once rather than several times, but you might decide differently - I don't think it matters too much